api/pom.xml

```diff
@@ -40,7 +40,7 @@
         <findbugs.exclude>exclude.xml</findbugs.exclude>
         <findbugs.threshold>Low</findbugs.threshold>
     </properties>
-    <name>jaxr-api</name>
+    <name>${extension.name} API</name>
```

Comment:
What is this "name" referring to?  How is it used?  If it's for the name of the repo, which it looks like, then this needs to stay as "jaxr-api".

Reply:
I just copied what was done in management-api. See line 34 above this:

<extension.name>javax.xml.registry</extension.name>

Reply:
Okay.  It doesn't look like it's being used anyway...